packages/core/src/ProgressBar/ProgressBar.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Flex, Box } from 'pcln-design-system'
+
+const CustomBox = styled(Box)`
+  width: 100%;
+  border-radius: 2px;
+  margin-right: 4px;
+`
+
+const defaultStepColor = 'background.light'
+
+function ProgressBar({ steps, stepIndex, stepHeight }) {
+  return (
+    <Flex>
+      {steps.map((step, index) => {
+        const stepColor =
+          index < stepIndex ? steps[stepIndex - 1].color : defaultStepColor
+        return (
+          <CustomBox
+            key={step.color}
```

Comment:
Is the `step.color` definitely going to be unique? Would it make sense to use `index` since you're using that for `data-testid` anyway?

Reply:
Yeah that makes more sense probably, using index in this case when things can't move around is fine.